subsys/mgmt/ec_host_cmd/Kconfig

```diff
@@ -9,20 +9,39 @@ config EC_HOST_CMD
 	bool "Support Embedded Controller host command handler subsystem"
 	help
 	  Enable host command processing for embedded controllers on notebook
-	  computers. Enabling this option requires specifying a chosen
-	  zephyr,ec-host-interface device as the ec host command backend that
-	  receive incoming host command requests to process.
+	  computers. Only version 3 is supported.
```

Comment:
Drop "Only version 3 is supported" because this is noted in the documentation.

Reply:
Done.